test/tests/syntax.json

```diff
@@ -490,6 +490,31 @@
       "src": "{0E-1}",
       "exp": "0E-1"
     },
+    {
+      "description": "- as unquoted-literal",
+      "src": "{-}",
+      "exp": "-"
+    },
+    {
+      "description": ". as unquoted-literal",
+      "src": "{·}",
+      "exp": "·"
+    },
+    {
+      "description": "emoji as unquoted-literal",
+      "src": "{🥔}",
+      "exp": "🥔"
+    },
+    {
+      "description": "multi-glyph emoji as unquoted-literal",
+      "src": "{🏳️‍🌈}",
+      "exp": "🏳️‍🌈"
+    },
+    {
+      "description": "various characters as unquoted-literal",
+      "src": "{\u00a1\u061d\u1681\u200b\u2010\u2030\u2060\u206a\u3001\ue000\ufdf0}",
```

Comment:
Interesting to include some plane 1 and 2 characters?

Reply:
I just picked the first characters of each of the separate ranges beyond Latin-1 that are representable with JSON `\u` escapes.

Reply:
You can use JSON \u escapes--by employing a surrogate pair. But it's not important.

Reply:
You could have left your mark with the Finnish flag ;-)

Reply:
Sure, but 🥔 tastes better.